terraform/gcp/security/gcp-compute-boot-disk-encryption.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+- id: gcp-compute-boot-disk-encryption
+  patterns:
+    - pattern: |
+        resource "google_compute_instance" $ANYTHING {
+          ...
+        }
+    - pattern-not-inside: |
+        resource "google_compute_instance" $ANYTHING {
+          ...
+          boot_disk {
+            disk_encryption_key_raw = ...

```

Comment:
Are there other encryption methods besides raw keys? Something like the CMK keys in some of the other rules?

Reply:
According to [google_compute_instance](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance) terraform docs `kms_key_self_link` as well.